aws/s3/ingestBills.go

```diff
@@ -89,6 +90,10 @@ func UpdateDueReports(ctx context.Context, tx *sql.Tx) ([]ReportUpdateConclusion
 			}
 			aas[br.AwsAccountId] = aa
 		}
+		err = es.CleanCurrentMonthBillByBillRepositoryId(ctx, aa.UserId, br.Id)
+		if err != nil {
+			l.Info("No incomplete bills found for this bill repository.", err)
```

Comment:
Does ES return an error if there's nothing to delete ? If it doesn't, `Error when cleaning the current month.` is more appropriate as well as Error instead of Info.

Please add the account `aa` and the bill repository `br` to the log.

Reply:
It return an error when there aren't any element to delete.